### ISSUE TITLE
YAML comment at start of R code chunk for alt text

### ISSUE THREAD

**Issue by joshwlambert** [r-lib/pkgdown#2867]
Thanks for the work on making {pkgdown} maximise the accessibility of a package documentation! 

After spotting the warnings that alt text is missing from figures when running `pkgdown::build_site()`, I tried to fix it using the `#| fig.alt: >` syntax (as shown in `vignette(pkgdown::accessibility)`). However, it took me some time to realise, AFAICS, that the YAML commands have to be at the beginning of the code chunk. I think it would be good to mention this in the `accessibility` vignette to hopefully prevent others from stumbling on this.

**Comment by jayhesselberth** [comment]
Sorry this tripped you up. But I think the example in the accessibility vignette is pretty clear:

````{verbatim}
```{r}
#| fig.alt: >
#|   Histogram of time between eruptions for Old Faithful. 
#|   It is a bimodal distribution with peaks at 50-55 and 
#|   80-90 minutes.
hist(faithful$waiting)
```
````
